server/lomas_server/tests/test_api.py

```diff
@@ -36,6 +42,9 @@
 INITAL_EPSILON = 10
 INITIAL_DELTA = 0.005
 
+enable_features("floating-point")
```

Comment:
do we need to call this on the server somewhere? I suppose enabling it on the client side also enables it on the server side if we run local tests (all is in the same python environment I guess).

Reply:
Good point, I did not think about this. 
Just now I think it means that it is not 'peer-reviewed' so it is 'less safe' . Maybe we should discuss with Raphaël about this.